examples/typescript-context/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "ava-typescript-context",
+	"description": "TypeScript example for typing t.context",
+	"scripts": {
+		"test": "ava"
+	},
+	"devDependencies": {
+		"@sindresorhus/tsconfig": "^1.0.2",
+		"ava": "^3.15.0",
+		"ts-node": "^10.0.0",
+		"typescript": "^4.3.4"
+	},
+	"ava": {
+		"extensions": [
+			"ts"
+		],
+		"require": [
+			"ts-node/register"
+		]
```

Comment:
Can we make this work with `@ava/typescript`? Perhaps the built-in compiler mode that's in a pre-release?

Reply:
Done! I totally forgot about `@ava/typescript`, my bad! This is much better. Also the `compile` flag seems to work with version 2 :).